src/EditorFeatures/Text/Extensions.SnapshotSourceText.cs

```diff
@@ -82,8 +80,8 @@ public static SourceText From(Workspace workspace, ITextSnapshot editorSnapshot)
                 if (!s_textSnapshotMap.TryGetValue(editorSnapshot, out var snapshot))
                 {
                     // Avoid capturing `workspace` on the fast path
```

Comment:
Nit: comment name needs updating.

Reply:
I'll update this in a commit in my follow-up pull request to avoid needing to rebuild everything again.